### PR TITLE
[13.0][MIG] account_move_line_tax_editable

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash administrative commits with the closer regular commit, and is this still needed?

**Comment by MiquelRForgeFlow** [comment]
> is this still needed?

I haven't found yet this feature implemented in Odoo.

**Comment by pedrobaeza** [comment]
I remember to see the fields in a standard runbot. Maybe when they are filled?

**Comment by MiquelRForgeFlow** [comment]
That's because:

In `account.view_move_line_form` of odoo v12 you find:

```xml
<group string="Taxes" attrs="{'invisible': [('tax_line_id','=',False), ('tax_ids','=',[])]}">
    <field name="tax_line_id" readonly="1" attrs="{'invisible': [('tax_line_id','=',False)]}"/>
    <field name="tax_ids" widget="many2many_tags" readonly="1" attrs="{'invisible': [('tax_ids','=',[])]}"/>
</group>
```

In `account.view_move_line_form` of odoo v13 you find:

```xml
<group string="Taxes" attrs="{'invisible': [('tax_line_id','=',False), ('tax_ids','=',[])]}">
    <field name="tax_line_id" readonly="1" attrs="{'invisible': [('tax_line_id','=',False)]}"/>
    <field name="tax_ids" widget="many2many_tags" readonly="1" attrs="{'invisible': [('tax_ids','=',[])]}"/>
    <field name="tax_audit"/>
</group>
```

But this module amends this by using the `is_tax_editable` field. The behavior is the same as v12, that hasn't changed.

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Migration commit --> [MIG]

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

File: account_move_line_tax_editable/views/account_move.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//field[@name='line_ids']/tree/field[@name='name']" position="after">
+            <xpath expr="//field[@name='invoice_line_ids']/tree/field[@name='name']" position="after">
```
Comment (pedrobaeza) [inline account_move_line_tax_editable/views/account_move.xml]: Invoice lines? This should be on journal items.

Comment (MiquelRForgeFlow) [reply account_move_line_tax_editable/views/account_move.xml]: Well, in v13 there are two pages, one for invoice_line_ids and another for line_ids. I checked that the old page of line_ids in v12 is the invoice_line_ids for v13 (comparing the fields in the page). The one page new is for the line_ids, that show a different list of fields.

Comment (pedrobaeza) [reply account_move_line_tax_editable/views/account_move.xml]: They are in an incorrect position:

Selección_026

And this is only valid when you are looking to journal items, not to invoices. Even more, I would say that they should be hidden if the move is an invoice.

Comment (MiquelRForgeFlow) [reply account_move_line_tax_editable/views/account_move.xml]: haha ok

Comment (MiquelRForgeFlow) [reply account_move_line_tax_editable/views/account_move.xml]: can you check now?

**Review by HaraldPanten — APPROVED** [review]
Functional review. LGTM 👍

**Pull request merged**